rosdep/python.yaml

```diff
@@ -3944,6 +3944,21 @@ python-transforms3d-pip:
   ubuntu:
     pip:
       packages: [transforms3d]
+python-transitions:
```

Comment:
can you please use the wildcard syntax for the current distributions? this will allow to not have to add manually new ubuntu distributions (e.g. cosmic that has this package as well). 

```yaml
  debian:
    '*': [python-transitions]
    stretch:
      pip:
        packages: [transitions]
  ubuntu:
    '*': [python-transitions]
    trusty:
      pip:
        packages: [transitions]
    xenial:
      pip:
        packages: [transitions]
```